go.mod

```diff
@@ -1,8 +1,8 @@
-module github.com/deathowl/go-metrics-prometheus
+module github.com/subchord/go-metrics-prometheus
 
 go 1.13
 
 require (
-	github.com/prometheus/client_golang v1.1.0
-	github.com/rcrowley/go-metrics v0.0.0-20190826022208-cac0b30c2563
+	github.com/prometheus/client_golang v1.2.1
+	github.com/subchord/go-metrics v1.0.1
```

Comment:
Hi @SubChord, we should specify the version, but I think we should keep the original repository `github.com/rcrowley/go-metrics`

Reply:
I agree but the rcrowley/go-metrics hasn't been updated in months. It does not support the distribution metric.

Reply:
Sorry I dont want to depend on a Fork. Please get rcrowley to merge your pr, and then I can merge yours.